### DISCUSSION TITLE
Google Search for apps

### DISCUSSION THREAD

**Discussion by ponarangan** [airsdk/Adobe-Runtime-Support#3157] [General]
What are the ways to bring my app in Google search results. Is there any ANE that I should integrate?

**Comment by marchbold** [comment]
Can you elaborate some more on what you are looking for? 

I'm not aware of any native SDK functionality that affects search results.

**Comment by ponarangan** [comment]
When I search for certain games on Firefox or Chrome, some apps are displayed in the search results. I want to know whether I should add any extensions to make my app appear in Google search results on the web browsers.

**Comment by ponarangan** [comment]
![Untitled-2](https://github.com/airsdk/Adobe-Runtime-Support/assets/90509957/16e1e1cc-f9f2-4fce-9238-75d397320ca7)
Example

**Reply by GeneralVimes** [reply]
It's just SEO, search engine optimization. Make sure to have links to your game with texts. Such links can be from your website, from Reddit (from those subreddits, which allow content creators post own stuff), from Youtube: make a video about your game and add a link to the description.

It would be good to post your game to the appropriate showcases: https://airsdk.dev/showcase https://wiki.starling-framework.org/games/start

Create a page on Itch.io for your game and place links to the mobile builds there, etc.
